com.unity.cinemachine/Documentation~/samples-tutorials.md

```diff
@@ -2,15 +2,48 @@
+| **Custom Blends** | Third-person follow with character switching. | <ul> <li>Set up a custom camera blending algorithm.</li> <li>Map the custom algorithm to a Cinemachine Brain event.</li> </ul> |
+| **Cutscene** | Third-person follow with treasure area. | <ul> <li>Set up a cutscene with camera blends and asset animations in Timeline.</li> <li>Trigger a cutscene that blends in from the game camera, plays through, and then blends back out to the game camera.</li> </ul> |
+| **Fly around** | First-person flight simulator. | <ul> <li>Set up a fly-around camera with basic height and speed controls. |
+| **FreeLook Deoccluder** | Third-person free look with obstacle handling. | <ul> <li>Set up a free look camera that handles occlusion by walls to keep the player in view.</li> <li>Set up the scene and the camera to consider certain objects as transparent and ignore them in the occlusion evaluation.</li> </ul> |
+| **FreeLook on Spherical Surface** | Third-person free look with local up direction. | <ul> <li>Set up a free look camera to follow a player that walks on the surface of a sphere.</li> <li>Set up the camera to follow the character either lazily or actively.</li> </ul> |
+| **Impulse Wave** | Third-person follow and impulse wave effect. | <ul> <li>Set up the scene and the camera to make them react to impulse waves.</li> <li>Invoke an impulse from a fixed epicenter.</li> <li>Trigger an impulse when the player jumps.</li> </ul> |
+| **Lock-on Target** | Third-person follow with boss area. | <ul> <li>Set up a simple third-person free look camera to look at the player and rotate around it with the mouse.</li> <li>Set up a camera that looks at the player and locks on a boss character when the player enters a trigger zone.</li> </ul>  |
+| **Mixing Camera** | Car racing game with stunt. | <ul> <li>Set up a camera group that continuously blends multiple cameras to reflect the car speed.</li> <li>Set up a fixed camera that activates for a cut-in when the car enters a two-ramp stunt zone, to frame the car jump from the side.</li> </ul> |
+| **Running Race** | Character race game (simulation). | <ul> <li>Set up a group of cameras that each follow a different player.</li> <li>Get the cameras automatically switch to always have the leader of the race centered on screen.</li> <li>Set up an on-demand camera that frames all runners while looking at the sun.</li> <li>Emulate players that move along predefined paths.</li> </ul>  |
+| **Split Screen Car** | Local multiplayer car racing game. | <ul> <li>Display two racing cars in a split screen configuration.</li> </ul> |
+| **Third Person With Aim Mode** | Third-person shooter. | <ul> <li>Set up a simple free look camera that follows the player as it moves, jumps, and sprints.</li> <li>Add noise to the camera to emulate a handheld effect.</li> <li>Set up an on-demand camera that allows the player to aim and fire projectiles precisely with a crosshair despite the handheld effect.</li> </ul> |
```

Comment:
This is the opposite of a simple free look camera!  This is a vey important sample that illustrates many things - in particular the paradigm of rotating the player aiming core to drive the camera position.  There is deep co-operation between the player controller (sample code is provided) and the camera.  This ensures precise synchronization between character and camera, allowing superior control over aiming and occlusion handling.  This is in contrast to a simple FreeLook camera which manages its own orbit.

Reply:
Ok, the free look concept is one of the things that still jams in my mind. I'm going to meet you soon if you have time so that we can clarify together. For the rest, that's very valuable info (thanks a lot!) but I still need to sort it out to provide it at best in the top-down way. Definitely, we could expand the bullet list.